app/ssttparrangement/ArrangementController.scala

```diff
@@ -312,7 +312,7 @@ class ArrangementController @Inject() (
     val schedule = calculatorService.computeSchedule(journey)
 
     val initialPayment = if (schedule.initialPayment > exact(0)) Some(schedule.initialPayment.toString()) else None
-    val initialStartDate = initialPayment.fold[Option[LocalDate]](None)(_ => Some(schedule.startDate.plusWeeks(1)))
+    val initialStartDate = initialPayment.fold[Option[LocalDate]](None)(_ => Some(schedule.startDate.plusDays(calculatorService.defaultInitialPaymentDays)))
```

Comment:
I was wondering if the `PaymentSchedule` case class should have an `initialPaymentDate` field? 

That'd mean it's then all down to the CalculatorService to determine that date rather than also remembering to poke it in here.

Reply:
Yes, that makes sense what you say.  If you have time, feel free to bring it in next pr :)